weblate/addons/properties.py

```diff
@@ -136,6 +139,8 @@ class PropertiesSortAddon(BaseAddon):
     description = gettext_lazy("Formats and sorts the Java properties file.")
     compat = {"file_format": {"properties-utf8", "properties", "gwt"}}
     icon = "sort-alphabetical.svg"
+    settings_form = PropertiesSortAddonForm
 
     def pre_commit(self, translation, author) -> None:
-        format_file(translation.get_filename())
+        case_sensitive = self.instance.configuration["case_sensitive"]
```

Comment:
Make sure this works also with the old configurations, which didn't have this key:

```suggestion
        case_sensitive = self.instance.configuration.get("case_sensitive", False)
```

Reply:
A good catch @nijel. This is helpful to avoid issues when Weblate is updated to the version with this change.